src/components/snapshot/overview/SnapshotOverview.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { RouteComponentProps } from 'react-router-dom';
+import { connect, useDispatch } from 'react-redux';
+import { Typography } from '@mui/material';
+import { createStyles, WithStyles, withStyles } from '@mui/styles';
+import { TdrState } from '../../../reducers';
+import { BreadcrumbType, SnapshotIncludeOptions } from '../../../constants';
+import { useOnMount } from '../../../libs/utils';
+import { getSnapshotById, getSnapshotPolicy, getUserSnapshotRoles } from '../../../actions';
+import { PolicyModel, SnapshotModel } from '../../../generated/tdr';
+import AppBreadcrumbs from '../../AppBreadcrumbs/AppBreadcrumbs';
+import SnapshotOverviewPanel from './SnapshotOverviewPanel';
+import SnapshotRelationshipsPanel from '../../common/overview/SchemaPanel';
+
+const styles = () =>
+  createStyles({
+    pageRoot: {
+      padding: '16px 24px',
+      display: 'flex',
+      flexDirection: 'column',
+      height: '100%',
+    },
+    root: {
+      // TODO: expect this to change as more components are added
+      height: '100%',
+      display: 'grid',
+      gridTemplateColumns: '1fr 3fr',
+      flex: 1,
+    },
+    infoColumn: {
+      display: 'flex',
+      flexDirection: 'column',
+    },
+    infoColumnPanel: {
+      flexGrow: 1,
+    },
+    mainColumn: {
+      display: 'flex',
+      flexDirection: 'column',
+      marginLeft: 40,
+    },
+    pageTitle: {
+      marginBottom: '1rem',
+    },
+  });
+
+interface RouteParams {
+  uuid: string;
+}
+
+type AllSnapshotProps = StateProps & WithStyles<typeof styles> & RouteComponentProps<RouteParams>;
+
+function SnapshotOverview(props: AllSnapshotProps) {
+  const { classes, match, snapshot, snapshotPolicies } = props;
+  const snapshotId = match.params.uuid;
+  const dispatch = useDispatch();
```

Comment:
@nmalfroy I can get `dispatch` directly in a component or define it as a component [property](https://github.com/DataBiosphere/jade-data-repo-ui/blob/2028e65f9f8e3535f7aca588c2f46f7f10c7892f/src/components/snapshot/overview/SnapshotExport.tsx#L42) (I'm not sure where that gets passed in though). Is one method preferred over the other?